Jellyfin.Api/Controllers/LyricController.cs

```diff
@@ -0,0 +1,248 @@
+    public async Task<ActionResult> DeleteLyrics(
+        [FromRoute, Required] Guid itemId)
+    {
+        var audio = _libraryManager.GetItemById<Audio>(itemId);
+        if (audio is null)
+        {
+            return NotFound();
+        }
+
+        await _lyricManager.DeleteLyricsAsync(audio).ConfigureAwait(false);
+        return NoContent();
+    }
+
+    /// <summary>
+    /// Search remote lyrics.
+    /// </summary>
+    /// <param name="itemId">The item id.</param>
+    /// <response code="200">lyrics retrieved.</response>
+    /// <returns>An array of <see cref="RemoteLyricInfo"/>.</returns>
+    [HttpGet("Items/{itemId}/RemoteSearch/Lyrics")]
+    [Authorize(Policy = Policies.LyricManagement)]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public async Task<ActionResult<IEnumerable<RemoteLyricInfo>>> SearchRemotelyrics([FromRoute, Required] Guid itemId)
+    {
+        var audio = _libraryManager.GetItemById<Audio>(itemId);
+        if (audio is null)
+        {
+            return NotFound();
+        }
+
+        return await _lyricManager.SearchLyricsAsync(audio, false, CancellationToken.None).ConfigureAwait(false);
+    }
+
+    /// <summary>
+    /// Downloads a remote lyric.
+    /// </summary>
+    /// <param name="itemId">The item id.</param>
+    /// <param name="lyricId">The lyric id.</param>
+    /// <response code="204">Lyric downloaded.</response>
+    /// <returns>A <see cref="NoContentResult"/>.</returns>
+    [HttpPost("Items/{itemId}/RemoteSearch/Lyrics/{lyricId}")]
+    [Authorize(Policy = Policies.LyricManagement)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    public async Task<ActionResult> DownloadRemoteLyrics(
+        [FromRoute, Required] Guid itemId,
+        [FromRoute, Required] string lyricId)
+    {
+        var audio = _libraryManager.GetItemById<Audio>(itemId);
+        if (audio is null)
+        {
+            return NotFound();
+        }
+
+        try
+        {
+            await _lyricManager.DownloadLyricsAsync(audio, lyricId, CancellationToken.None).ConfigureAwait(false);
+
+            _providerManager.QueueRefresh(audio.Id, new MetadataRefreshOptions(new DirectoryService(_fileSystem)), RefreshPriority.High);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Error downloading lyrics");
+        }
+
+        return NoContent();
+    }
+
+    /// <summary>
+    /// Gets the remote lyrics.
+    /// </summary>
+    /// <param name="id">The remote provider item id.</param>
+    /// <response code="200">File returned.</response>
+    /// <returns>A <see cref="FileStreamResult"/> with the lyric file.</returns>
+    [HttpGet("Providers/Lyrics/{id}")]
+    [Authorize(Policy = Policies.LyricManagement)]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesFile(MediaTypeNames.Text.Plain)]
+    public async Task<ActionResult> GetRemoteLyrics([FromRoute, Required] string id)
```

Comment:
```suggestion
    public async Task<ActionResult> GetRemoteLyrics([FromRoute, Required] string lyricId)
```

More consistent with downloadRemoteLyrics and indicative of what id we want